tox.ini

```diff
@@ -3,14 +3,16 @@
 
 [tox]
 requires =
-    tox==4.6.0
+    tox>=4.6.0
 skipsdist = true
-envlist = py{38,39}, pylint, coverage
+envlist = py{38,39,310,311}, pylint, coverage
 
 [gh-actions]
 python =
     3.8: py38, pylint, coverage
     3.9: py39
+    3.10: py310
```

Comment:
I did not verify it on my machine (where I have only up to 3.9.19 installed), so I am taking this as meaning that both 3.10 and 3.11 are tested in CI, correct?

I can validate that all changes made above are still working with 3.9.19.

Reply:
Yes, 3.10 and 3.11 are tested in CI now as well